contrib/document_cleanup/README.md

```diff
@@ -0,0 +1,16 @@
+# Document Image Cleanup
```

Comment:
The two-folder structure seems a bit redundant. Why not remove the parent folder "document_cleanup" and this readme? The "light_weight_document_cleanup_ICDAR2021" folder has a readme with the same images also.